<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trainer_registration_windowClass</class>
 <widget class="QDialog" name="Trainer_registration_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainer_registration_window</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>301</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #EBE9F5;
border-radius: 20px;

</string>
   </property>
   <widget class="QLabel" name="password_status_tr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input_tr">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input_tr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QToolButton" name="register_button_tr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_status_tr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryCombo_tr">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAfterCurrent</enum>
    </property>
    <property name="placeholderText">
     <string>Category</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Fitnnes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Body_building</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="login_input_tr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname_input_tr">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>124</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLabel" name="surname_label_alert_tr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_label_alert_tr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="password_label_alert_tr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="login_label_alert_tr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trainer registration</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/background.png)</string>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
